File: src/main/java/com/gfi/bin/admctasweb/operativos/model/ExclusionModel.java
package com.gfi.bin.admctasweb.operativos.model;

public class ExclusionModel {
	private Integer idExclusion;
	private String descripcionExclusion;
	private Boolean activo;
	
	public Integer getIdExclusion() {
		return idExclusion;
	}
	public void setIdExclusion(Integer idExclusion) {
		this.idExclusion = idExclusion;
	}
	public String getDescripcionExclusion() {
		return descripcionExclusion;
	}
	public void setDescripcionExclusion(String descripcionExclusion) {
		this.descripcionExclusion = descripcionExclusion;
	}
	public Boolean getActivo() {
		return activo;
	}
	public void setActivo(Boolean activo) {
		this.activo = activo;
	}
}

File: src/main/webapp/js/gfi/bin/admctasweb/model/operativos/ExclusionModel.js
Ext.define('gfi.bin.admctasweb.model.operativos.ExclusionModel', {
	extend: 'Ext.data.Model',
	
	fields: [
	         {name	: 'idExclusion',	    	type	: 	'number'},
	         {name	: 'descripcionExclusion',	type	: 	'string'},
	         {name	: 'activo',   				type	: 	'boolean'}
	],
	
	validations: [
	         {type: 'presence', field: 'descripcionExclusion',	message: 'Debe introducir la descripcion'},
	         {type: 'length', field: 'descripcionExclusion', max:50, message: 'La longitud máxima para la descripcion es de 50'}
	]
});